src/main/java/net/minecraftforge/fml/relauncher/ReflectionHelper.java

```diff
@@ -28,8 +30,11 @@
 import java.lang.reflect.Constructor;
 import java.lang.reflect.Field;
 import java.lang.reflect.Method;
+
 /**
  * Some reflection helper code.
+ * Consider using {@link net.minecraftforge.fml.common.ObfuscationReflectionHelper}
+ * when dealing with obfuscated fields or methods.
```

Comment:
This is in FML's coremod land. Its split in two because the two classes need to exist in seperate classloaders. Modders shouldn't be using this class anyways.